app/domain_models/record_sets_update.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'common_error'
```

Comment:
I think update is not so necessary.
Update means delete and create at the same time.